com.github.wwmm.easyeffects.yml

```diff
@@ -0,0 +1,373 @@
+# see https://github.com/wwmm/easyeffects/blob/master/PKGBUILD for dependencies
+# If you get errors like 137, you might need pass --jobs=1 to flatpak-builder to limit cpu threads.
+  # discussion at here https://github.com/flathub/com.github.wwmm.pulseeffects/pull/53
+# todo
+# add x-checker-data
+# fix icon
```

Comment:
What icon do you need to fix?

Reply:
It's the icon when clicking the about page. See this screenshot: 
![image](https://user-images.githubusercontent.com/38842733/127210852-3603be9f-83c4-4592-974b-39d024e80406.png). As mentioned in the todolist in the github PR I want to report that upstream.

I think I should remove those comments from the manifest as they've been replaced by the PR todo list.

Reply:
Ok, so this is fixed now as of https://github.com/flathub/flathub/pull/2429/commits/6c5fa3b50d5fd4a4996b632310354218e9d2a0f4.

It's not a perfect fix, as ideally we don't patch these things.

I'm not sure what the ideal fix is here. Should upstream EasyEffects follow Flatpak's favoured reverse-domain convention? The [rename-icon docs suggests an in tree change is ideal](https://docs.flatpak.org/en/latest/manifests.html?highlight=rename-icon#file-renaming), but I'm not sure if distro packages would like that.